src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports

```diff
@@ -0,0 +1,12 @@
+net.devh.boot.grpc.server.autoconfigure.GrpcAdviceAutoConfiguration
```

Comment:
Do you really need all of this to make it work?

Reply:
yes. read comment above.